hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/TestHoodieBackedMetadata.java

```diff
@@ -1737,6 +1739,75 @@ public void testErrorCases() throws Exception {
     }
   }
 
+  /**
+   * Tests no more than 1 clean is scheduled/executed if HoodieCompactionConfig.allowMultipleCleanSchedule config is disabled.
+   */
+  @Test
+  public void testMultiClean() throws Exception {
```

Comment:
Is there any file which has cleaner specific tests? This does not seem metadata table related test.